### ISSUE TITLE
Fatal Error related to memory access - closes after 3-5 seconds (Windows 11)

### ISSUE THREAD

**Issue by rbutera** [FancyWM/fancywm#109]
If you can fix this for me, I **promise** I will buy a license.

I am using FancyWM on a fresh install of windows 11.

It was working fine for about an hour, but I did a restart and now I get an application error every time about 3 seconds after I load FancyWM

**FancyWM continues to work while the dialog option is open!**

As soon as I click "OK" fancyWM closes.

I tried resetting the data, uninstalling, reinstalling, nothing works.

## To Reproduce
Unknown

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Windows 11
 - Version [e.g. 22H2 64-bit]

**I have UAC disabled - maybe that's relevant? will try with UAC enabled.**

### Log:

```
2022-07-03 17:58:21.930 +01:00 [WRN] FancyWM v2.6.7.0 (https://www.microsoft.com/store/apps/9P1741LKHQS9) on Microsoft Windows NT 10.0.22621.0
2022-07-03 17:58:22.966 +01:00 [WRN] Using the multi-monitor tiling backend
```

### Config:

```json
{
  "ActivationHotkey": "LWin_LShiftKey",
  "ShowStartupWindow": false,
  "AllocateNewPanelSpace": true,
  "AnimateWindowMovement": true,
  "WindowPadding": 4,
  "PanelHeight": 18,
  "PanelFontSize": 12,
  "ShowFocusDuringAction": true,
  "OverrideAccentColor": false,
  "CustomAccentColor": "#0000FFFF",
  "Keybindings": {
    "ToggleManager": {
      "IsDirectMode": false,
      "Keys": [
        "F11"
      ]
    },
    "PullWindowUp": {
      "IsDirectMode": false,
      "Keys": [
        "Return"
      ]
    },
    "ToggleFloatingMode": {
      "IsDirectMode": false,
      "Keys": [
        "F"
      ]
    },
    "CreateHorizontalPanel": {
      "IsDirectMode": false,
      "Keys": [
        "H"
      ]
    },
    "CreateVerticalPanel": {
      "IsDirectMode": false,
      "Keys": [
        "V"
      ]
    },
    "CreateStackPanel": {
      "IsDirectMode": false,
      "Keys": [
        "S"
      ]
    },
    "RefreshWorkspace": {
      "IsDirectMode": false,
      "Keys": [
        "F5"
      ]
    },
    "MoveFocusLeft": {
      "IsDirectMode": false,
      "Keys": [
        "Left"
      ]
    },
    "MoveFocusUp": {
      "IsDirectMode": false,
      "Keys": [
        "Up"
      ]
    },
    "MoveFocusRight": {
      "IsDirectMode": false,
      "Keys": [
        "Right"
      ]
    },
    "MoveFocusDown": {
      "IsDirectMode": false,
      "Keys": [
        "Down"
      ]
    },
    "MoveLeft": {
      "IsDirectMode": false,
      "Keys": [
        "LeftCtrl",
        "Left"
      ]
    },
    "MoveUp": {
      "IsDirectMode": false,
      "Keys": [
        "LeftCtrl",
        "Up"
      ]
    },
    "MoveRight": {
      "IsDirectMode": false,
      "Keys": [
        "LeftCtrl",
        "Right"
      ]
    },
    "MoveDown": {
      "IsDirectMode": false,
      "Keys": [
        "LeftCtrl",
        "Down"
      ]
    },
    "SwapLeft": {
      "IsDirectMode": false,
      "Keys": [
        "LeftShift",
        "Left"
      ]
    },
    "SwapUp": {
      "IsDirectMode": false,
      "Keys": [
        "LeftShift",
        "Up"
      ]
    },
    "SwapRight": {
      "IsDirectMode": false,
      "Keys": [
        "LeftShift",
        "Right"
      ]
    },
    "SwapDown": {
      "IsDirectMode": false,
      "Keys": [
        "LeftShift",
        "Down"
      ]
    },
    "IncreaseWidth": {
      "IsDirectMode": false,
      "Keys": [
        "Oem6"
      ]
    },
    "IncreaseHeight": {
      "IsDirectMode": false,
      "Keys": [
        "OemQuotes"
      ]
    },
    "DecreaseWidth": {
      "IsDirectMode": false,
      "Keys": [
        "OemOpenBrackets"
      ]
    },
    "DecreaseHeight": {
      "IsDirectMode": false,
      "Keys": [
        "Oem1"
      ]
    },
    "Cancel": {
      "IsDirectMode": false,
      "Keys": [
        "Escape"
      ]
    },
    "SwitchToDesktop1": {
      "IsDirectMode": false,
      "Keys": [
        "D1"
      ]
    },
    "SwitchToDesktop2": {
      "IsDirectMode": false,
      "Keys": [
        "D2"
      ]
    },
    "SwitchToDesktop3": {
      "IsDirectMode": false,
      "Keys": [
        "D3"
      ]
    },
    "SwitchToDesktop4": {
      "IsDirectMode": false,
      "Keys": [
        "D4"
      ]
    },
    "SwitchToDesktop5": {
      "IsDirectMode": false,
      "Keys": [
        "D5"
      ]
    },
    "SwitchToDesktop6": {
      "IsDirectMode": false,
      "Keys": [
        "D6"
      ]
    },
    "SwitchToDesktop7": {
      "IsDirectMode": false,
      "Keys": [
        "D7"
      ]
    },
    "SwitchToDesktop8": {
      "IsDirectMode": false,
      "Keys": [
        "D8"
      ]
    },
    "SwitchToDesktop9": {
      "IsDirectMode": false,
      "Keys": [
        "D9"
      ]
    },
    "MoveToDesktop1": {
      "IsDirectMode": false,
      "Keys": [
        "LeftShift",
        "D1"
      ]
    },
    "MoveToDesktop2": {
      "IsDirectMode": false,
      "Keys": [
        "LeftShift",
        "D2"
      ]
    },
    "MoveToDesktop3": {
      "IsDirectMode": false,
      "Keys": [
        "LeftShift",
        "D3"
      ]
    },
    "MoveToDesktop4": {
      "IsDirectMode": false,
      "Keys": [
        "LeftShift",
        "D4"
      ]
    },
    "MoveToDesktop5": {
      "IsDirectMode": false,
      "Keys": [
        "LeftShift",
        "D5"
      ]
    },
    "MoveToDesktop6": {
      "IsDirectMode": false,
      "Keys": [
        "LeftShift",
        "D6"
      ]
    },
    "MoveToDesktop7": {
      "IsDirectMode": false,
      "Keys": [
        "LeftShift",
        "D7"
      ]
    },
    "MoveToDesktop8": {
      "IsDirectMode": false,
      "Keys": [
        "LeftShift",
        "D8"
      ]
    },
    "MoveToDesktop9": {
      "IsDirectMode": false,
      "Keys": [
        "LeftShift",
        "D9"
      ]
    }
  },
  "ProcessIgnoreList": [
    "Taskmgr"
  ],
  "ClassIgnoreList": [
    "OperationStatusWindow"
  ],
  "RemindToRateReview": true,
  "ShowContextHints": true,
  "MultiMonitorSupport": true,
  "SoundOnFailure": true
}
```

**Comment by rbutera** [comment]
Just re-enabled UAC and still the same error, different instruction address

**Comment by grahfmusic** [comment]
I also suffer from the same problem - I'm on Windows 11 Developers Edition at 5120x1440 resolution.

**Comment by veselink1** [comment]
Thanks for opening a bug report. Not much can be done without an error message. 

Some users experienced a "Win32Exception: Not enough quota is available to process this command." error in v2.6.7.0. This has largely been fixed as of v2.8.x.

@grahfmusic Insider builds of Windows are not guaranteed to be compatible

**Comment by rbutera** [comment]
You need an error?

Okay all it says is "an instruction at address X attempted to access memory at address 0"

Basically it's a generic segfault
